### DISCUSSION TITLE
How can I translate this magick command to Magick.NET?

### DISCUSSION THREAD

**Discussion by trustieee** [dlemstra/Magick.NET#1266] [Help]
I'm having trouble translating this magick command to the Magick.NET api, specifically the sizing parts.
Any help is appreciated, thanks!

```
magick tiff1.tiff -gravity southwest -background white -bordercolor none -splice 0x2.5% \
-size %[fx:w*0.1]x ( label:"TEXT HERE" -border 10% ) -composite out.jpg
```

**Comment by dlemstra** [comment]
The sizing part is done through the fx method. But in C# you could just do `image.Width * 0.1`?

**Comment by trustieee** [comment]
Are you able to translate the rest of the command? I'm not seeing a translation for something like `-border 10%` in the Magick.NET API

**Comment by dlemstra** [comment]
Sorry for the late response but for those who are finding this post through searches. `-border 10%` is the same as `image.Border(image.Width*10, image.Height*10)` but in the next release of Magick.NET you will be able to do this: `image.Border(new Percentage(10))`.
